datafusion/proto/src/logical_plan/to_proto.rs

```diff
@@ -1623,6 +1625,35 @@ impl From<JoinConstraint> for protobuf::JoinConstraint {
     }
 }
 
+impl From<Constraints> for protobuf::Constraints {
```

Comment:
I didn't see a test for this code, perhaps you can add one to https://github.com/apache/arrow-datafusion/blob/main/datafusion/proto/tests/cases/roundtrip_logical_plan.rs so we have some basic coverage

Reply:
Added roundtrip tests for protobuf. Thanks for the suggestion.